<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>615</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>975</width>
    <height>615</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>975</width>
    <height>615</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ACL bahosi uchun dastur</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>971</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tanlanma olish</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>251</width>
        <height>551</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label11">
         <property name="text">
          <string>Bosh to'plam taqsimoti:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox11">
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label12">
         <property name="text">
          <string>Parametrlar:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="5" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox15">
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label16">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label13">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label14">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label17">
           <property name="text">
            <string>Surilish pram.:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox11">
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="maximum">
            <double>10000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label15">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox12">
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="maximum">
            <double>10000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label18">
           <property name="text">
            <string>Masshtab param.:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox14">
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox16">
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox13">
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label19">
           <property name="text">
            <string>Tanlanma hajmi:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox11">
           <property name="minimum">
            <number>100</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label110">
           <property name="text">
            <string>Senzurlanish soni:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox12">
           <property name="maximum">
            <number>80</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton11">
         <property name="text">
          <string>Bajar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget11">
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton12">
         <property name="text">
          <string>Tanlanmani saqlash</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>31</width>
        <height>551</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>651</width>
        <height>551</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>ACL bahosi</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>641</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton21">
         <property name="text">
          <string>Tanlanma yuklash</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton22">
         <property name="text">
          <string>Hisoblash</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton23">
         <property name="text">
          <string>Natijalarni saqlash</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="tableWidget21">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>251</width>
        <height>551</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line_6">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>31</width>
        <height>551</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>50</y>
        <width>651</width>
        <height>491</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>ACL bahosi uchun kriteriy</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>921</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="comboBox31"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton31">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Tanlanma yuklash</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton32">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Kriteryni hisoblash</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton33">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Natijalarni saqlash</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit31">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>641</width>
        <height>491</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="tabletTracking">
       <bool>true</bool>
      </property>
      <property name="documentTitle">
       <string>xcczxcxz</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget31">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>71</y>
        <width>251</width>
        <height>491</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>31</width>
        <height>491</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>975</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFayl">
    <property name="title">
     <string>Fayl</string>
    </property>
    <addaction name="actionOchish_2"/>
    <addaction name="actionSaqlash_2"/>
    <addaction name="actionSaqlash_3"/>
   </widget>
   <widget class="QMenu" name="menuKo_rinish">
    <property name="title">
     <string>Ko'rinish</string>
    </property>
   </widget>
   <addaction name="menuFayl"/>
   <addaction name="menuKo_rinish"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSaqlash">
   <property name="text">
    <string>Saqlash</string>
   </property>
  </action>
  <action name="actionOchish_2">
   <property name="text">
    <string>Ochish</string>
   </property>
  </action>
  <action name="actionSaqlash_2">
   <property name="text">
    <string>Saqlash</string>
   </property>
  </action>
  <action name="actionSaqlash_3">
   <property name="text">
    <string>Saqlash...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
